test/cypress/e2e/home.cy.ts

```diff
@@ -7,7 +7,7 @@ describe('Home Page', () => {
 
       cy.visit('/');
 
-      const homePageHeroText = 'Discover the Casper Blockchain.';
+      const homePageHeroText = 'Discover the Jenkins blockchain.';
```

Comment:
Jenkins eh?

Reply:
Yeah, I'm not sure who Jenkins is.

Reply:
Check your .env file. This is set under ORG_NAME

Reply:
Ahhh ^ that'll do it!